package.json

```diff
@@ -60,8 +62,11 @@
     "cypress": "^7.3.0",
     "cypress-file-upload": "^5.0.3",
     "cypress-real-events": "^1.3.0",
-    "express": "^4.14.0",
```

Comment:
Do we really need to move express from devDepenencies to dependencies?

Reply:
For Heroku, we have to. I'll move to devDependicies after resolving issues

Reply:
Moved express back to devDependicies